tests/wpt/mozilla/tests/mozilla/like.html

```diff
@@ -0,0 +1,314 @@
+<!doctype html>
```

Comment:
If we make this file a `.any.js` then it will run in worker scopes as well (https://web-platform-tests.org/writing-tests/testharness.html#tests-for-other-or-multiple-globals-any-js)

Reply:
done